clouddriver-dcos/src/main/groovy/com/netflix/spinnaker/clouddriver/dcos/deploy/util/mapper/DeployDcosServerGroupDescriptionToAppMapper.groovy

```diff
@@ -24,261 +24,252 @@ import mesosphere.marathon.client.model.v2.*
 import static com.google.common.base.Strings.emptyToNull
 
 class DeployDcosServerGroupDescriptionToAppMapper {
-  App map(final String resolvedAppName, final DeployDcosServerGroupDescription description) {
```

Comment:
Can't quite tell with the change to the indentation exactly what's been changed.